src/alr/alr-commands-init.adb

```diff
@@ -63,6 +63,8 @@ package body Alr.Commands.Init is
          Filename : constant String :=
             +Full_Name (Directory / (+Lower_Name & ".gpr"));
       begin
+         --  Use more than 80 colums for more readable strings
+         pragma Style_Checks ("M200");
```

Comment:
I'm going to copy this trick at some point...

Reply:
We should avoid it :)